### DISCUSSION TITLE
custom background size

### DISCUSSION THREAD

**Discussion by roughnecks** [JulianPrieber/littlelink-custom#800] [Questions]
Hello,

what's the recommended size (width/height) for custom backgrounds in themes?

Thanks

**Comment by JulianPrieber** [comment]
Pretty much anything goes. I usually go with 16:9, that works well on desktop and can be scaled to mobile (just cropped).
I make sure to compress any images used, I go with resolutions up to 1080p.

**Comment by roughnecks** [comment]
Thanks.

I used a portrait background which has an image on the left side of the page only and it works "quite" fine in mobile too, only a glitch on my old phone. Then following your advice I modified it to be fullhd, 16:9 and in mobile, even for tablets (using chrome developer console), the image on the left just vanishes..

So what's your advice, go with landscape and accept the loss on mobile or keep using my portrait image? Or perhaps doing some other changes to landscape to make it work..

**Reply by JulianPrieber** [reply]
You can center the image using CSS.
There are quite a few options you can do to get your image displayed fully, but due to the resolution difference, there will always be some type of compromise you'll have to make.

Here is some documentation that could help: https://www.w3schools.com/cssref/pr_background-image.php
